snippets/native-platforms/ionic2/use.md

```diff
@@ -1,98 +1,142 @@
 ```js
-// src/services/auth/auth.service.ts
+// src/services/auth.service.ts
 
-import { Storage } from '@ionic/storage';
-import { AuthHttp, JwtHelper, tokenNotExpired } from 'angular2-jwt';
+import { JwtHelper, tokenNotExpired } from 'angular2-jwt';
 import { Injectable, NgZone } from '@angular/core';
-import { Observable } from 'rxjs/Rx';
+import { Observable, Subscription } from 'rxjs';
+import { Auth0Vars } from '../auth0-variables';
+
+import Auth0Cordova from '@auth0/cordova';
+import Auth0 from 'auth0-js';
+
+const auth0Config = {
+  // needed for auth0
+  clientID: Auth0Vars.AUTH0_CLIENT_ID,
+
+  // needed for auth0cordova
+  clientId: Auth0Vars.AUTH0_CLIENT_ID,
```

Comment:
we can replace the values for these vars with the actual values for the client. let's do this:
```
clientId: '${account.clientId}',
domain: '${account.namespace}',
callbackURL: 'http://localhost:8200',
packageIdentifier: 'YOUR-PACKAGE-ID'
```